app/services/WcoMetadataMapping.scala

```diff
@@ -15,31 +15,96 @@
  */
 
 package services
-import forms.declaration.{Document, PreviousDocumentsData}
+import forms.declaration._
 import models.DeclarationFormats._
 import models.declaration.SupplementaryDeclarationData
+import services.Countries.allCountries
 import uk.gov.hmrc.http.cache.client.CacheMap
-import uk.gov.hmrc.wco.dec.{Declaration, GovernmentAgencyGoodsItem, MetaData, PreviousDocument}
-
+import uk.gov.hmrc.wco.dec.{
```

Comment:
Hmm, I'm thinking about import with `_`

`import uk.gov.hmrc.wco.dec._`

Reply:
that would be too big of a package